pnpm-lock.yaml

```diff
@@ -1,270 +1,406 @@
-lockfileVersion: '9.0'
```

Comment:
Looks like you are using older version of pnpm. Please use latest one.

Reply:
Sorry, fixed!!
https://github.com/yukukotani/eslint-plugin-chakra-ui/pull/223/commits/4ea2eb78ae5c599c9b183bf3f718917d19816619